html/app.js

```diff
@@ -10,6 +10,25 @@ import {AudioChunk, CompressedAudioChunk, PlaceholderChunk, concat_chunks, Clock
 //   by forcing this to load up top, before we try to 'addModule' it.
 import './audio-worklet.js';
 
+addEventListener('error', (event) => {
+  if (document.getElementById('crash').style.display) {
+    return;
```

Comment:
I wonder whether we should have a list of these, instead of only showing the first one?